src/componentsUniversal/button/button.js

```diff
@@ -1,8 +1,8 @@
 import './Button.scss'
 
-const Button = ({text, className=''}) => {
+const Button = ({text, className='', onClick}) => {
     return (
-        <button  className={`btn ${className}`}>{text}</button>
+        <button  className={`btn ${className}`} onClick={onClick}>{text}</button>
```

Comment:
you can skip return and brackets

const Button = ({ text, className = '', onClick }) => (
    <button className={`btn ${className}`} onClick={onClick}>{text}</button>
)